.idea/jarRepositories.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We shouldn't be committing any of these .idea files. Please update the PR by removing these files